metricbeat/module/kibana/stats/_meta/settings_data.json

```diff
@@ -0,0 +1,55 @@
+{
+    "@timestamp": "2020-10-02T16:14:24.034Z",
+    "@metadata": {
+        "beat": "metricbeat",
+        "type": "_doc",
+        "version": "8.0.0"
+    },
+    "kibana": {
+        "stats": {
+            "settings": {
```

Comment:
Do we need to add field aliases from `kibana_settings.*` to `kibana.stats.settings.*`? @chrisronline are any of the `kibana_settings.*` fields used in aggregations/filtering or are they just read from source? Even if it's the latter, is the UI prepared to alternatively read from `kibana.stats.settings.*` fields with this PR?

Reply:
No field aliasing necessary.

>  is the UI prepared to alternatively read from kibana.stats.settings.* fields with this PR?

I don't think we actually read from this currently. I think the only consumer is the ES cluster alert watches, which read them from source. However, I think it will be moot as those [should be removed in 7.12](https://github.com/elastic/kibana/issues/81020)